joomla/joomla-cms
# [4.0] Move the additional language strings in a separate file & fix postinstall

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/18301#issuecomment-360113436

### Summary of Changes

> Is there a reason why all the language strings are in the .sys.ini language file and not spread across two language files as normal

Thanks @brianteeman 

- Fixing the postinstall behavior as this was broken before.

### Testing Instructions

- install this branch https://github.com/zero-24/joomla-cms/archive/httpheader.zip
- Check the config page of the httpheader plugin and make sure all language strings are still loaded. -
- Check the postinstall message and make sure all language strings are still loaded too. (please make sure to disable the plugin inorder to trigger the postinstall message)

### Expected result

two files & postinstall works also on new installs

### Actual result

one file & postinstall is broken.
